### PR TITLE
Using conda packages from ISMRMRD org

### PR DESCRIPTION
**Pull request by hansenms**
This PR pulls in the packages in the [ISMRMRD conda org](https://anaconda.org/ismrmrd), since they are now produced directly from the respective repos. Reving the minor version too, since this could be breaking in some setups. 

Note particularly that due to changes in `siemens_to_ismrmrd` we need to update some reference data for tests.

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -49,31 +49,31 @@ requirements:
...
-    - cuda-runtime=11.6.1            # [linux64]
+    - cuda-libraries=11.6.1               # [linux64]
+    - cuda-runtime=11.6.1                 # [linux64]
     - dcmtk=3.6.1
     - fftw=3.3.9
-    - gadgetron-python>=1.3.6
-    - ismrmrd=1.7.1
-    - ismrmrd-python>=1.9.8
+    - gadgetron::gadgetron-python>=1.3.6
+    - ismrmrd::ismrmrd=1.12.1
```
Comment (roopchansinghv) [inline conda/meta.yaml:57]: This should probably be:

   - ismrmrd::ismrmrd=1.11.1

**Comment by hansenms** [comment]
Due to the changes in https://github.com/ismrmrd/siemens_to_ismrmrd/commit/6bf0ca0273e8af621034563be25a5057eb758b82 there are some table positions that are now incorrect. We will need to update the test cases for:

* cases/cmr_cine_binning.cfg
* cases/cmr_cine_binning_2slices.cfg
* cases/cmr_cine_binning_2slices_distributed.cfg
* cases/generic_cartesian_cine_denoise.cfg
* cases/generic_grappa_T2W.cfg
* cases/generic_grappa_snr_R2_ipat_PEFOV75_PERes75_PEOverSampling120.cfg
* cases/generic_grappa_tse.cfg
* cases/generic_nl_spirit_cartesian_sampling_cine.cfg
* cases/generic_spirit_cartesian_sampling_spat2.cfg

**Comment by xueh2** [comment]
I will update the ref data and put up a PR, on Monday.

**Comment by hansenms** [comment]
Thanks @xueh2, I have updated the test data, maybe you can have a look at see that I didn't goof anything up?

**Review by xueh2 — APPROVED** [review]
Looks all right to me.

**Pull request merged**